### PR TITLE
[Core] Bump to OpenAssetIO-alpha.14

### LINKED ISSUE #4: Update template to OpenAssetIO v1.0.0-alpha.14
Various breakages due to

- https://github.com/OpenAssetIO/OpenAssetIO/issues/1048
- https://github.com/OpenAssetIO/OpenAssetIO/issues/1054

Remember to check docs.

### PR DESCRIPTION
**Pull request by feltech**
Also bump OpenAssetIO-MediaCreation for good measure, though it's not technically necessary

Closes #4

### CODE REVIEW

File: plugin/my_asset_manager/MyAssetManagerInterface.py [hunk 1]
```diff
@@ -61,24 +62,24 @@ def info(self):
...
-            # supported In this case, if any of the input traitSets are
+            # The host asks specifically if sets of traits are
+            # supported. In this case, if any of the input traitSets are
             # for read, and contain LocatableContent, as we can supply
             # data for that trait, we imbue a managed policy response,
             # as well as the traits we are able to supply data for. It's
             # important to get this right, for more info, see:
-            # https://openassetio.github.io/OpenAssetIO/classopenassetio_1_1v1_1_1host_api_1_1_manager.html#acdf7d0c3cef98cce7abaf8fb5f004354
-            if context.isForRead() and LocatableContentTrait.kId in traitSet:
+            # https://openassetio.github.io/OpenAssetIO/classopenassetio_1_1v1_1_1manager_api_1_1_manager_interface.html#ab86b5623a355d04086bae76875ebee17
```
Comment (elliotcmorris) [inline plugin/my_asset_manager/MyAssetManagerInterface.py:81]: [Question] It's unfortunate that these hashes are based on the parameters of the method, as it makes linking to them unstable if we make changes. You think there's any way to link in a more stable manner? (I cant find anything.)
Good catch btw.

File: plugin/my_asset_manager/MyAssetManagerInterface.py [hunk 2]
```diff
@@ -61,24 +62,24 @@ def info(self):
...
-    def managementPolicy(self, traitSets, context, hostSession):
+    def managementPolicy(self, traitSets, policyAccess, context, hostSession):
         # The management policy defines which traits the manager is
-        # capable of imbuing queried traitSets with. In this case, we
-        # manage locations of assets only, and only in a read, not a
-        # write, context. Note `LocatableContentTrait` is a trait from
-        # the openassetio-mediacreation library, see :
+        # capable of imbuing queried traitSets with. In this case, the
+        # only workflow we support is to resolve the current location of
+        # pre-existing assets. Note `LocatableContentTrait` is a trait
```
Comment (elliotcmorris) [inline plugin/my_asset_manager/MyAssetManagerInterface.py]: [Consider][Minor] 
I think this rephrasing makes it slightly more difficult to connect to what's going on, as we don't use the terminology. (ie, read/access)
```suggestion
        # manager allows read access to the locations of assets. 
        # Note `LocatableContentTrait` is a trait
```
"pre-existing" also seems a tad redundant.

**Review by elliotcmorris — APPROVED** [review]
:+1: 

One opinionated doc nitpick and one lamentation.

File: plugin/my_asset_manager/MyAssetManagerInterface.py (hunk 1 above)
Comment (feltech) [reply plugin/my_asset_manager/MyAssetManagerInterface.py:81]: I had a quick non-exhaustive google but didn't find a good solution :disappointed:

File: plugin/my_asset_manager/MyAssetManagerInterface.py (hunk 2 above)
Comment (feltech) [reply plugin/my_asset_manager/MyAssetManagerInterface.py]: Yeah I like your suggestion, squashed it in.

**Comment by elliotcmorris** [comment]
Merge when ready.

**Pull request merged**